Debian/openvas/debian/shlibs.local

```diff
@@ -0,0 +1,6 @@
+libgvm_base 21 gvm-libs
+libgvm_gmp 21 gvm-libs
+libgvm_osp 21 gvm-libs
+libgvm_util 21 gvm-libs
+libgvm_boreas 21 openvas
```

Comment:
`libgvm_boreas` and `libgnutls` are dependent shared libraries for OpenVas 21.4 version.